apps/src/dance/ai/utils.ts

```diff
@@ -0,0 +1,29 @@
+import {BlockSvg, Workspace} from 'blockly';
+
+/**
+ * Generates blocks from the AI result in a given workspace,
+ * and attaches them to each other.
+ */
+export const generateBlocksFromResult = (
```

Comment:
Thanks for moving this out into a separate file! How do you feel about just adding this to `dance/utils.ts` and renaming something like `generateAIBlocksFromResult` or something? That way we just have one utils file, though I can see the benefit of having an AI-specific utils file.

Reply:
Looking at other existing `utils` files, in `p5lab` directory there's a `utils.js` and in `p5lab/poetry/commands`, there's another `utils.js`.

To me it feels nice to have this file in `dance/ai` since it is so ai block-specific.

Reply:
No strong preference!